wcfsetup/install/files/lib/system/user/authentication/password/algorithm/Invalid.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
Note: Add deprecation notice for the `invalid()` implementation in `PasswordUtil`.